tools/diagnostics/create_service_diagnostics_bundle.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+readonly DCOS_COMMONS_SCRIPT_PATH="tools/diagnostics"
+
+function is_dcos_commons_repository_script() {
+  [[ ${SCRIPT_DIRECTORY} = *${DCOS_COMMONS_SCRIPT_PATH} ]]
+}
+
+function is_development_mode() {
+  is_dcos_commons_repository_script
+}
+
+if is_development_mode; then
+  echo "dcos-commons repository detected, running in development mode"
+  echo
+  echo "In development mode all Python modules will be picked up from your current"
+  echo "dcos-commons repository instead of the Docker image's /dcos-commons-dist "
+  echo "directory that contains a static git checkout"
+  echo
+  set -x
+  readonly CONTAINER_DCOS_COMMONS_DIRECTORY="/dcos-commons"
+  readonly CONTAINER_DCOS_COMMONS_VOLUME_MOUNT="-v $(pwd):${CONTAINER_DCOS_COMMONS_DIRECTORY}:ro"
+else
+  readonly CONTAINER_DCOS_COMMONS_DIRECTORY="/dcos-commons-dist"
+  # We don't mount the dcos-commons directory in the container because the
+  # script will use /dcos-commons-dist which is added to the Docker image during
+  # build time.
+  readonly CONTAINER_DCOS_COMMONS_VOLUME_MOUNT=
+fi
+
+readonly VERSION='v0.1.0'
+
+function version () {
+  echo "${VERSION}"
+}
+
+if [ "${#}" -eq 1 ] && [[ "${1}" =~ ^(--version|-version|version|--v|-v)$ ]]; then
+  version
+  exit 0
+fi
+
+readonly REQUIREMENTS='docker'
+
+for requirement in ${REQUIREMENTS}; do
+  if ! [[ -x $(command -v "${requirement}") ]]; then
+    echo "You need to install '${requirement}' to run this script"
+    exit 1
+  fi
+done
+
+readonly BUNDLES_DIRECTORY="service-diagnostic-bundles"
+readonly DOCKER_IMAGE="mpereira/dcos-commons:diagnostics-${VERSION}"
```

Comment:
We're still using your personal `docker` image.

Reply:
This is gonna be addressed in https://jira.mesosphere.com/browse/DCOS-42310

Reply:
I think we should do it now in the following steps:
1. Change the image to `mesosphere/dcos-commons:diagnostics-${VERSION}`
1. Build the docker image off this branch
1. Merge!
(This would also address the ticket you mention above)

In the long run we would need to think a bit about what it meant to "release" this script.

Reply:
But I'm OK to merge this if you feel that we should handle all docker-related issues in the linked JIRA ticket.